### PR TITLE
Task sequence revert

### PR DESCRIPTION
**Pull request by aejjehint**
This reverts PR #12453 and #13080

The original PR introducing task sequence was checked in prematurely as the backend support isn't in yet. When the backend support is in we will re-submit this change.

### CODE REVIEW

**Review by premanandrao — APPROVED** [review]

**Review by smanna12 — APPROVED** [review]

**Review by mdtoguchi — APPROVED** [review]

**Review by AlexeySachkov — APPROVED** [review]
Would be good to understand motivation for the revert - did we decide to drop the feature, or did it cause some regressions which are hard to debug?

**Review by steffenlarsen — APPROVED** [review]

**Comment by GarveyJoe** [comment]
I agree with Alexey, please include in the description the reasoning for the revert: i.e.:

"This change was checked in prematurely as the backend support isn't in yet.  When the backend support is in we will re-submit this change."

**Review by GarveyJoe — COMMENTED** [review]
commented

**Comment by aejjehint** [comment]
@AlexeySachkov @GarveyJoe I added to the description the reason behind the revert.

**Review by GarveyJoe — APPROVED** [review]

File: sycl/include/sycl/device_aspect_macros.hpp [hunk 1]
```diff
@@ -691,11 +686,6 @@
 #define __SYCL_ANY_DEVICE_HAS_ext_oneapi_graph__ 0
 #endif
 
-#ifndef __SYCL_ANY_DEVICE_HAS_ext_intel_fpga_task_sequence__
-// __SYCL_ASPECT(ext_intel_fpga_task_sequence__, 62)
-#define __SYCL_ANY_DEVICE_HAS_ext_intel_fpga_task_sequence__ 0
-#endif
-
 #ifndef __SYCL_ANY_DEVICE_HAS_ext_oneapi_limited_graph__
 // __SYCL_ASPECT(ext_oneapi_limited_graph, 63)
```
Comment (sommerlukas) [inline sycl/include/sycl/device_aspect_macros.hpp:690]: This and the following defines should all have been updated, similar to `ALL_DEVICE_HAS`.

The ID specified for `ext_oneapi_limited_graph` now does not match the ID defined in `aspects.def` anymore. The same applies for all following defines.

@aejjehint Can you please fix this in a follow-up PR?

**Pull request merged**